tests/publiccloud/run_ltp.pm

```diff
@@ -0,0 +1,147 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Use perl script to run LTP on public cloud
+#
+# Maintainer: Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base "publiccloud::basetest";
+use strict;
+use testapi;
+use utils;
+use serial_terminal 'select_virtio_console';
+use Data::Dumper;
+
+sub assert_ssh
```

Comment:
1. it is not assert_ssh, looks more like execute_over_ssh
2. I think you can fully replace it with select_console('root-ssh') . Don't get me wrong I am not asking you to drop ltp-ssh backend I just asking you to drop this function